docs/tutorials/upgrading-ev3dev-using-shell

```diff
@@ -0,0 +1,24 @@
+---
+title: Upgrading ev3dev Using Shell
+subject: Upgrading
+---
+
+Once you've written the ev3dev image to the SD card, booted, and setup a network 
+connection, you may want to update to the latest kernel.  To do so:
```

Comment:
It would be nice to make a note here that you are also upgrading the drivers, so there may be changes that break user software. Maybe include a link to the news feed (`/news`, I think) and say that one should make sure that they are aware of breaking changes before upgrading.